src/gameProtocol.js

```diff
@@ -0,0 +1,21 @@
+import { GAME_STATUS } from './constants.js'
+
+const X = 1
+const O = 0
+const EMPTY = 9
+
+export const mockGameStateEventArgs = {
+  id: '',
+  players: [
+    { id: '', username: '', piece: X },
+    { id: '', username: '', piece: O }
+  ],
+  playerTurn: '',
+  state: GAME_STATUS.STARTED,
+  board: [EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY]
+}
+
+export const mockMovePlayedEventArgs = {
+  player: '',
```

Comment:
After discussions about issues with the game protocol, a decision has been made to add the `gameId` property to **MOVE_PLAYED** event. This would allow simplify sending **GAME_STATE** and persisting game state
```js
const eventArgs ={
      gameId: 'game-uuid',
      player: 'player-uuid',
      move: { piece: 1 , position: 5 } // 'piece' is either 0 or 1 , 'position' is a number from 0 to 8 (based on the 'board' array)
     //   const X = 1;
     //   const O = 0;
 }
```